src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+  const result = {};
+
+  sourceArray.forEach((elem) => {
+    const newElem = elem.trim();
+    const arr = newElem.split(':');
+
+    if (arr.length > 1) {
+      result[arr[0].trim()] = arr[1].trim();
+    }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You could replace the `forEach` with a `reduce` to build the result object in a more functional programming style.